examples/blog-service/README.md

```diff
@@ -1,11 +1,117 @@
+
+### Blog post submission
+ 
+When a blog post is submitted, the system attempts to determine the most appropriate category (from a predefined list) and assign a rating between 1 and 10. If a suitable category is found and the rating exceeds a defined threshold (e.g., 3), the blog post is accepted and persisted along with the identified category and rating.
+
+- The implementation uses an LLM to identify the most suitable category and a rating based on the specified review criteria. This is implemented using natural expressions in Ballerina, specifically a [natural function](./review_blog.bal) with the requirement specified in natural language. Ballerina's natural expressions handle incorporating the response schema in the request to the LLM and binding the response to the expected type automatically. 
+
+    ```ballerina
+    # Represents a blog entry with title and content.
+    public type Blog record {|
+        # The title of the blog
+        string title;
+        # The content of the blog
+        string content;
+    |};
+
+    # Review of a blog entry.
+    type Review record {|
+        # Suggested category
+        string? suggestedCategory;
+        # Rating out of 10
+        int rating;
+    |};
+
+    public isolated function reviewBlog(Blog blog) returns Review|error => natural {
+        You are an expert content reviewer for a blog site that 
+            categorizes posts under the following categories: ${categories}
+
+            Your tasks are:
+            1. Suggest a suitable category for the blog from exactly the specified categories. 
+            If there is no match, use null.
+
+            2. Rate the blog post on a scale of 1 to 10 based on the following criteria:
+            - **Relevance**: How well the content aligns with the chosen category.
+            - **Depth**: The level of detail and insight in the content.
+            - **Clarity**: How easy it is to read and understand.
+            - **Originality**: Whether the content introduces fresh perspectives or ideas.
+            - **Language Quality**: Grammar, spelling, and overall writing quality.
+
+            Here is the blog post content:
+
+            Title: ${blog.title}
+            Content: ${blog.content}
+    };
+    ```
+
+- The rest of the logic, including accepting or rejecting the blog depending on the category and/or rating, persisting the data, etc. are all handled entirely in Ballerina.
+
+### Blog post retrieval
+
+Blog posts can be retrieved by category and are automatically sorted by the rating assigned during submission. This retrieval functionality is also implemented entirely in Ballerina.
+
+## Building and running the sample
+
+Ballerina Swan Lake Update 13 (2201.13.0) will include experimental support for natural expressions. Try out natural expressions today by installing a milestone release of 2201.13.0.
```

Comment:
I think we needs to add a link to the guide in here for install U13 milestones

Reply:
That's covered in step 1 - Set up prerequisites to use natural expressions.